package.json

```diff
@@ -31,28 +34,24 @@
   ],
   "scripts": {
     "audit": "lerna run audit --stream",
-    "bootstrap": "lerna bootstrap",
-    "bump": "lerna publish",
     "commit": "cz",
     "doctoc": "doctoc README.md packages/collector/README.md",
     "lint": "node misc/is_eol || (eslint .*.js *.js && npm run typecheck && lerna run lint --stream && eslint misc && eslint dockerfile-examples && eslint bin)",
-    "postinstall": "husky install && npm run bootstrap",
+    "postinstall": "husky install && lerna run node_modules:exists",
```

Comment:
FYI: npm workspaces is very optimized to install the dependencies across the project. For some packages we do not have dependencies which will be installed in the target folder. That's why it can happen that a package does not have a `node_modules` folder. But we need a `node_modules` folder for our core/metrics package. The metrics package [determines](https://github.com/instana/nodejs/blob/v2.33.1/packages/core/src/util/applicationUnderMonitoring.js#L217) the app package.json based on the existence of the `node_modules` folder. I am wondering if should remove this check.